packages/block-editor/package.json

```diff
@@ -66,6 +67,7 @@
 		"@wordpress/wordcount": "file:../wordcount",
 		"change-case": "^4.1.2",
 		"classnames": "^2.3.1",
+		"clipboard": "2.0.11",
```

Comment:
This is a new dependency.

Reply:
In other places, we just use `event.clipboardData.setData`